core/src/i18n/en.yaml

```
@@ -37,6 +37,7 @@ common:
     delete-confirm: Don't ask this again
     message: Are you sure you want to remove {{type}} {{name}}?
     title: Remove {{name}}
+    information: Delete confirmations will be disabled for all resources. It can be turned back on in Preferences.
```

```suggestion
    information: You've switched off delete confirmations for all resources. You can switch them back on in **Preferences**.
```
I suggest active voice and no future tense. Not sure whether we can/want to use bold type here, thus only a suggestion, not editing myself.